reference/constraints.rst

```diff
@@ -42,6 +42,8 @@ Validation Constraints Reference
    constraints/DateTime
    constraints/Time
 
+   constraints/Timezone
```

Comment:
why not moving it directly after Time?

Reply:
Because `Date`, `DateTime` & `Time` constraints validate date and time representations while `Timezone` validates a timezone string. So to me, the inner meaning is different.